### PR TITLE
wiki: Add Redmi Note 12 5G/Poco X5 5g (stone)

### PR DESCRIPTION
**Pull request by Nomishaw21**

### CODE REVIEW

**Review by Nomishaw21 — COMMENTED** [review]
Yes

File: docs/official_devices/list.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ sidebar_class_name: hidden
 ### Xiaomi
 - [POCO F3 / Mi 11x / Redmi K40 (alioth)](/devices/alioth)
 - [POCO M3 Pro/Redmi Note 10T/10 5G/11 SE (camellia)](/devices/camellia)
+- [POCO x5 5g / Redmi note 12 5g (stone)](/devices/stone)
```
Comment (neroices) [inline docs/official_devices/list.md:30]: Why are you adding the device to discontinued support?
Also, please use proper naming (capitalization, etc.)

Here's my suggestion:
```
- [POCO X5 5G / Redmi Note 12 5G(stone)](/devices/stone)
```

File: docs/official_devices/xiaomi/stone.md
Comment (neroices) [inline docs/official_devices/xiaomi/stone.md:1]: It seems you're using GNU Nano as your editor

File: docs/official_devices/xiaomi/stone.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+slug: /devices/stone
+pagination_next: null
+pagination_prev: null
+title: "Redmi Note 12 5G / POCO X5 5G (stone)*"
```
Comment (neroices) [inline docs/official_devices/xiaomi/stone.md:5]:
```
title: "POCO X5 5G / Redmi Note 12 5G(stone)"
```

File: docs/official_devices/xiaomi/stone.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+<a href="https://www.pling.com/p/2058150/" class="button button-->
+
+## Installation Guide
+:::caution
+Make sure to backup your data before proceeding.
+:::
+
+### Installing Recovery
+1. Enter fastboot mode by using a key combination `Power + Vol Do>
```
Comment (neroices) [inline docs/official_devices/xiaomi/stone.md:33]: *Vol Down.

File: docs/official_devices/xiaomi/stone.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+2. Connect your device to your PC via USB.
+3. Verify that your PC detects the device with `fastboot devices`.
+4. Flash the boot image onto your device using `fastboot flash bo>
```
Comment (neroices) [inline docs/official_devices/xiaomi/stone.md:36]:
```
`fastboot flash boot recovery.img`.
```

File: docs/official_devices/xiaomi/stone.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+2. Reboot into recovery mode.
+3. Select Apply update > Apply from ADB.
+6. Now you can start sideloading by this command:
+```
+adb sideload halcyon_stone-xxxxx.zip
+```
+
+## Troubleshooting
+
+If you encounter any issues during or after the installation, fee>
```
Comment (neroices) [inline docs/official_devices/xiaomi/stone.md:50]: *feel free to ask to our chat group for solutions to common problems.

**Review by neroices — CHANGES_REQUESTED** [review]

**Pull request merged**